### ISSUE TITLE
feature-branch: deprecation-notice

### ISSUE THREAD

**Issue by AiyionPrime** [freifunk/meshviewer#10]
I'm not sure if upstream would be interested in the deprecation notice; I assume it has already been tried to contribute it back but was declined?

**edit**: consider the next portion nevermind

> 
> If so, any chance we could open a feature-branch in this repo, to collaboratively work on the deprecation list without having to filter the relevant commits to cherry-pick each time we rebuild our map
> 
> I think the feature is a really worthwhile addition but maintaining it locally will becoming an utter pain. I heard @grische plans on updating their copy of the list in the next days as well.
> Maybe we can focus our efforts here?
> 
> **edit**
> 
> I think everybody here knows feature branches; but for others:
> One would open a branch which contains every commit that touches the deprecation-list and only that.
> The goal would to have this branch to be able to merge into your local repo without much trouble.

**Comment by AiyionPrime** [comment]
fml. Not sure how I overlooked https://github.com/freifunk-ffm/meshviewer/tree/feature_deprecation

Is that branch currently maintained? If so, I should reopen the PR against that one, right?

**Comment by margau** [comment]
From my Site, nothing in that fork is actively maintained, sorry.

**Comment by MyIgel** [comment]
The ffffm fork is actually "actively" maintained but not the branch itself as it was integrated in our main branch.
We could create a PR (and feel free to do that if you want) but as any contributions by anyone where very actively blocked by the upstream maintainer we gave up contributing there.

**Comment by AiyionPrime** [comment]
I'll prepare the branch then.

**Comment by AiyionPrime** [comment]
All right. I think I'm done.

https://github.com/freifunkh/meshviewer/tree/feature-deprecation-warning

This branch contains the commits @margau had in his original PR, plus some linting fixes so it can build using `gulp` again, as well as a change as upstream does not have the same config file as you do.

I'd suggest we put that branch in your meshviewer repo, open a PR against ffrgb and ask xaver to just let it stay there.
That way, we could easily track whether it still applies to upstream/master.

Whenever we then want to change the list of deprecated devices or the feature in general we can open a PR against the featurebranch, so that the PRs are taking place here and not in ffrgb's repo, so xaver doesn't have to bother with us at all.

**Comment by MyIgel** [comment]
Hmm, that sounds interesting :D

**Comment by weeman1337** [comment]
Most of the deprecations should be fixed by https://github.com/freifunk/meshviewer/issues/10 
There will still some be left depending on upstream https://github.com/freifunk/meshviewer/issues/49

**Comment by maurerle** [comment]
I think this can be closed..?
